src/averageAges.js

```diff
@@ -55,10 +63,52 @@ function calculateWomenAverageAge(people, withChildren) {
  * @return {number}
  */
 function calculateAverageAgeDiff(people, onlyWithSon) {
-  // 1. find a mother of each person (or only for men)
-  // 2. keep people who have mothers in the array
-  // 3. calculate the difference child.born - mother.born
-  // 4. return the average value
+  const children = people.filter((child) => {
+    return people
+      .some(mother => mother.name === child.mother)
```

Comment:
[2]. If you move your shortened return statement in arrow function to a newline, use round brackets. Besides, this return statement is a little bit hard to read, maybe it is better to make it simpler? For example, do you really need to split 69, 70 , 71 rows ? You can just write it all in one row